From 85c48ea3303d0ffe0afb74c033dd06373aad57f2 Mon Sep 17 00:00:00 2001
From: Tim Gates <tim.gates@iress.com>
Date: Mon, 9 Mar 2020 18:02:53 +1100
Subject: [PATCH 06/11] docs: Fix simple typo, proprt -> property

There is a small typo in treebeard/static/treebeard/treebeard-admin.js.

Should read `property` rather than `proprt`.
---
 treebeard/static/treebeard/treebeard-admin.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/treebeard/static/treebeard/treebeard-admin.js b/treebeard/static/treebeard/treebeard-admin.js
index b8daa6b..13c2f4c 100644
--- a/treebeard/static/treebeard/treebeard-admin.js
+++ b/treebeard/static/treebeard/treebeard-admin.js
@@ -41,7 +41,7 @@
                     var node = new Node(this);
                     node.collapse();
                 }).hide();
-                // Swicth class to set the proprt expand/collapse icon
+                // Swicth class to set the property expand/collapse icon
                 $elem.find('a.collapse').removeClass('expanded').addClass('collapsed');
             },
             parent_node: function () {
@@ -51,7 +51,7 @@
             expand: function () {
                 // Display each kid (will display in collapsed state)
                 this.children().show();
-                // Swicth class to set the proprt expand/collapse icon
+                // Swicth class to set the property expand/collapse icon
                 $elem.find('a.collapse').removeClass('collapsed').addClass('expanded');
 
             },
-- 
2.25.4

